lib/Test2/API.pm

```diff
@@ -110,7 +110,7 @@ use base 'Exporter';
 # This will make sure that Context.pm is loaded the first time this module is
 # imported, then the regular import method is swapped into place.
 sub import {
-    require Test2::API::Context unless $INC{'Test2/API/Context.pm'};
```

Comment:
Test2::API is only loaded once by Test::Builder, but it is the primary interface for all testing tools that are written against Test2, this optimization may matter when you consider that.